src/logout.ts

```diff
@@ -1,6 +1,6 @@
-Cypress.Commands.add('logout', ({ root, realm, redirect_uri }) =>
+Cypress.Commands.add('logout', ({ root, realm, redirect_uri, path_prefix = 'auth' }) =>
   cy.request({
     qs: { redirect_uri },
-    url: `${root}/auth/realms/${realm}/protocol/openid-connect/logout`,
+    url: `${root}${ path_prefix ? `/${path_prefix}` : ''}/realms/${realm}/protocol/openid-connect/logout`,
```

Comment:
Looks like the default value needs to be either `'/auth'`, or it shouldn't be removed from the url value.

I'm thinking the latter makes more sense. You'll always need the leading slash, right?

Reply:
According to the login, where I borrowed to code it should be fine:
https://github.com/babangsund/cypress-keycloak/blob/9044bad1dd2f07af04b9241ca4b1bf545fc4ed78/src/login.ts#L16-L18
with:
https://github.com/babangsund/cypress-keycloak/blob/9044bad1dd2f07af04b9241ca4b1bf545fc4ed78/src/login.ts#L12

Checking the syntax it should expand to:
`https://path.to.keycloak/$PATH_PREFIX/realms/...` if the pathprefix is not null
and
`https://path.to.keycloak/realms` if not set. At least `/${path_prefix}` should include the leading `/`
In any case you are right, that the leading slash is necessary.

I've tested the login using the value with `path_prefix: zauth` and it worked as expected.

Reply:
Okay, all good then. Thanks. :-)